src/main/java/teammates/ui/template/InstructorFeedbackSessionActions.java

```diff
@@ -58,7 +58,7 @@ public InstructorFeedbackSessionActions(PageData data, FeedbackSessionAttributes
                             Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS);
         }
 
-        this.isAllowedToSubmit = (session.isVisible() || session.isPrivateSession()) && shouldEnableSubmitLink;
+        this.isAllowedToSubmit = session.isOpened() && shouldEnableSubmitLink;
```

Comment:
There is a valid use case for allowing to submit before open time: to preview the questions. The condition used should be "(is visible or is private session) *and is not closed*".

Reply:
>The condition used should be "(is visible or is private session) and is not closed"

I implemented this at first, but as the use case you specified slipped my mind, I changed it to what it is right now, thus `this.isAllowedToSubmit = session.isOpened() && shouldEnableSubmitLink;`  . 

Making the changes.